src/dreyfus/src/clouseau_rpc.erl

```diff
@@ -78,16 +78,15 @@ group2(Ref, Args) ->
 delete(Ref, Id) ->
     rpc(Ref, {delete, couch_util:to_binary(Id)}).
 
+soft_delete(DbName) ->
+    rpc({main, clouseau()}, {soft_delete, DbName}).
```

Comment:
This should look like `rpc(Ref, {soft_delete, couch_util:to_binary(Id)}).` and be sent to the IndexService process itself (opening it if it's not already open in the usual manner).